apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -492,7 +492,7 @@ Keep in mind you are not just generating an example; you are generating an actua
   }
 
   job.data.html = html;
-  job.data.python = results.python;
+  job.data.python = python;
```

Comment:
if a python file *ever* existed, then this won't be null, so this fixed the disappearing python file problem.